src/Marten.FSharp/Marten.FSharp.fs

```diff
@@ -378,6 +378,9 @@ module Queryable =
         |> Lambda.ofArity1
         |> q.Count
 
+    let countAsync<'a> (q: IQueryable<'a>) = q.CountAsync()
+    let countAsyncWhere<'a> f (q: IQueryable<'a>) = q.CountAsync(Lambda.ofArity1 f)
```

Comment:
`countAsyncWhere` -> to `countTaskWhere` and see if there's a version that takes a CancellationToken and implemented a `countTaskWhereCt` version.